gprofiler/profilers/perf.py

```diff
@@ -37,6 +38,8 @@
 class PerfProcess:
     _dump_timeout_s = 5
     _poll_timeout_s = 5
+    _restart_after_s = 3600
+    _perf_memory_usage_treshhold = 536870912  # 512Mb
```

Comment:
```suggestion
    _perf_memory_usage_treshhold = 512 * 1024 * 1024
```

then, we don't need the comment :)